packages/espressocash_app/lib/features/activities/widgets/kyc_tile.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:flutter/material.dart';
+import 'package:kyc_client_dart/kyc_client_dart.dart';
+
+import '../../../di.dart';
+import '../../../l10n/l10n.dart';
+import '../../../ui/button.dart';
+import '../../../ui/colors.dart';
+import '../../kyc_sharing/services/kyc_service.dart';
+import '../../kyc_sharing/utils/kyc_utils.dart';
+import '../../kyc_sharing/widgets/kyc_status_icon.dart';
+import '../../ramp/partners/kyc/widgets/launch.dart';
+import '../../ramp_partner/models/ramp_type.dart';
+
+class EcKycTile extends StatelessWidget {
```

Comment:
Why `Ec` prefix? Why not just `KycTile`?

Reply:
updated